miasm2/expression/simplifications_common.py

```diff
@@ -250,6 +250,26 @@ def simp_cst_propagation(e_s, expr):
             e_s(Y.msb()) == ExprInt(0, 1)):
             args = [args[0].args[0], X + Y]
 
+    # ((var >> int1) << int2) => var & mask
```

Comment:
This is just a detail, but it would be great if the comment reflects `int1` and `int2` equality, that is to say something like "# ((var << int1) >> int1) => var & mask"